charts/kashti/templates/_helpers.tpl

```diff
@@ -36,7 +36,17 @@ Return the appropriate apiVersion for a networking object.
 {{- define "networking.apiVersion" -}}
 {{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
 {{- print "extensions/v1beta1" -}}
+{{- else if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- print "networking.k8s.io/v1" -}}
 {{- else -}}
 {{- print "networking.k8s.io/v1beta1" -}}
 {{- end -}}
```

Comment:
I'm being pickier than usual only because I know the pattern we settle on here is going to set a precedent that will end up being applied to a lot of our charts...

This logic is correct, but I think it will be easier to follow if it's rewritten as follows:

```
{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
{{- print "extensions/v1beta1" -}}
{{- else if semverCompare "<1.19-0" .Capabilities.KubeVersion.GitVersion -}}
{{- print "networking.k8s.io/v1beta1" -}}
{{- else -}}
{{- print "networking.k8s.io/v1" -}}
{{- end -}}
```

Reply:
This is the only remaining thing to address to get this merged.

Reply:
Valid point. Totally agree.